python/py-python-lsp-server/Portfile

```
@@ -0,0 +1,88 @@
+github.tarball_from archive
+revision            0
+
+categories-append   devel
+platforms           darwin
+supported_archs     noarch
+license             MIT
+maintainers         {reneeotten @reneeotten} openmaintainer
+
+description         A Python implementation of the Language Server Protocol
+long_description    Fork of the python-language-server project, maintained by the Spyder \
+                    IDE team and the community.
+
+checksums           rmd160  ed800ce49f963cd61cc239f3d1ecb8169af298ee \
+                    sha256  987eb87073fca65bd1b0d32e408935f2146a98d33ae06d716d00969fc8c935b3 \
+                    size    57851
+
+python.versions     36 37 38 39
+
+if {${subport} ne ${name}} {
+    depends_lib-append \
+                    port:py${python.version}-autopep8 \
+                    port:py${python.version}-codestyle \
+                    port:py${python.version}-docstyle \
+                    port:py${python.version}-flake8 \
+                    port:py${python.version}-flake8-mccabe \
+                    port:py${python.version}-jedi \
+                    port:py${python.version}-pluggy \
+                    port:py${python.version}-pyflakes \
+                    port:py${python.version}-pylint \
+                    port:py${python.version}-python-lsp-jsonrpc \
+                    port:py${python.version}-rope \
+                    port:py${python.version}-setuptools \
+                    port:py${python.version}-ujson \
+                    port:py${python.version}-yapf
+
+    patchfiles-append   patch-pylsp_plugins_flake8_lint.py.diff \
+                        patch-test_plugins_flake8_lint.py.diff
+
+    post-patch {
+        reinplace "s|@@FLAKE8_BIN@@|flake8-${python.branch}|g" ${worksrcpath}/pylsp/plugins/flake8_lint.py
+        reinplace "s|@@FLAKE8_BIN@@|flake8-${python.branch}|g" ${worksrcpath}/test/plugins/test_flake8_lint.py
+        reinplace "s|@@PYTHON_BIN@@|${python.bin}|g" ${worksrcpath}/pylsp/plugins/flake8_lint.py
+    }
+
+    depends_test-append \
+                    port:py${python.version}-pytest \
+                    port:py${python.version}-mock \
+                    port:py${python.version}-flaky
+
+    test.run        yes
+    test.cmd        py.test-${python.branch}
+    test.args       -o addopts=''
+    test.target
+    test.env        PYTHONPATH=${worksrcpath}/build/lib
+
+    post-destroot {
+        set docdir ${prefix}/share/doc/${subport}
+        xinstall -d ${destroot}${docdir}
+        xinstall -m 0644 -W ${worksrcpath} README.md LICENSE \
+            ${destroot}${docdir}
+    }
+
+    depends_run-append \
+                    port:python-lsp-server_select
+
+    select.group    python-lsp-server
+    select.file     ${filespath}/py${python.version}-pylsp
+
+    notes "
+To make the Python ${python.branch} version of python-lsp-server the one \
+that is run when you execute the commands without a version suffix, e.g. \
```

Avoid double spaces in notes output:
```suggestion
To make the Python ${python.branch} version of python-lsp-server the one\
that is run when you execute the commands without a version suffix, e.g.\
```